packages/storybook/src/web-components/alert/alert.stories.tsx

```diff
@@ -0,0 +1,240 @@
+import tokens from '@lux-design-system/design-tokens/dist/index.json';
+import {
+  LuxAlert as Alert,
+  LuxButton as Button,
+  LuxIconMail as IconMail,
+  LuxParagraph as Paragraph,
+} from '@lux-design-system/web-components-react';
+import type { JSX } from '@lux-design-system/web-components-stencil';
+import { useState } from '@storybook/preview-api';
+import type { Meta, StoryObj } from '@storybook/react';
+import { type HTMLAttributes, type PropsWithChildren } from 'react';
+import tokensDefinition from './tokens.json';
+import { createDesignTokensStory, createVisualRegressionStory, VisualRegressionWrapper } from '../../utils';
+
+const LuxAlert = (props: PropsWithChildren<JSX.LuxAlert> & HTMLAttributes<HTMLLuxButtonElement>) => (
+  <Alert {...props} />
+);
+LuxAlert.displayName = 'Test';
```

Comment:
Dit is ongebruikelijk dus zou er graag commentaar bij willen hebben over de waarom

Reply:
Dit was een test...